ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/FieldErrorTemplate.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import ListItem from "@material-ui/core/ListItem";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import List from "@material-ui/core/List";
+
+export default function FieldErrorTemplate(props) {
+  const { errors = [], idSchema } = props;
+  if (errors.length === 0) {
+    return null;
+  }
+  const id = `${idSchema.$id}__error`;
+
+  return (
+    <List dense={true} disablePadding={true}>
+      {errors.map((error, i) => {
+        return (error==="is a required property" ? null:
```

Comment:
@harkiratsm we never want to return this as it should be obvious from the red coloring of the field and is not otherwise informative or helpful to resolution for the user... right?

Reply:
actually with the current v5 migration there were some changes that left unchecked, one was this. So I added it, the point we talked about earlier with UX, on error is in pipeline would make appropriate accordingly. 2nd point at [#ref](https://app.clickup.com/t/2wnamcr)

Reply:
@harkiratsm in the meantime, though, please remove this verbiage. This is not desired.

Reply:
I am removing this template as it has no use as we are not actually modifying it.

Reply:
If require in future, I would add it back.